pandas/core/frame.py

```diff
@@ -5365,6 +5365,112 @@ def _construct_result(self, result) -> "DataFrame":
         out.columns = self.columns
         return out
 
+    def differences(self, other, axis=1, keep_indices=False, keep_values=False):
+        """
+        Compare to another DataFrame and show the differences.
```

Comment:
Can you place this docstring in the generic module and use the `Appender` / `Substitution` decorators to customize for the subclasses? Generally want to avoid copy / paste - will see other examples of how to do this in the module

Reply:
done